alf/trainers/on_policy_trainer.py

```diff
@@ -88,8 +88,8 @@ def train_():
             driver.run = tf.function(driver.run)
 
         env.reset()
-        time_step = driver.get_initial_time_step()
-        policy_state = driver.get_initial_state()
+        time_step = None
+        policy_state = None
```

Comment:
I assign the initial value for time_step and policy_state for the reason of keeping the calling signature to driver.run() same. tf.function will recompile the graph based on the calling signature.

Reply:
reverted